src/SFA.DAS.ApprenticeCommitments.Jobs.Functions/Mappings.cs

```diff
@@ -7,28 +7,28 @@ public static class Mappings
         public static ApprenticeshipCreated ToApprenticeshipCreated(this ApprenticeshipCreated2Event source) =>
             new ApprenticeshipCreated
             {
-                ApprenticeshipId = source.ApprenticeshipId,
+                CommitmentsApprenticeshipId = source.ApprenticeshipId,
                 Email = source.Email,
                 EmployerName = source.LegalEntityName,
                 EmployerAccountId = source.AccountId,
                 EmployerAccountLegalEntityId = source.AccountLegalEntityId,
                 TrainingProviderId = source.ProviderId,
-                AgreedOn = source.AgreedOn,
+                CommitmentsApprovedOn = source.CreatedOn
             };
 
         public static ApprenticeshipUpdated ToApprenticeshipUpdated(this ApprenticeshipCreated2Event source) =>
             new ApprenticeshipUpdated
             {
-                ContinuationOfCommitmentsApprenticeshipId = source.ContinuationOfId,
+                CommitmentsContinuedApprenticeshipId = source.ContinuationOfId,
                 CommitmentsApprenticeshipId = source.ApprenticeshipId,
-                CommitmentsApprovedOn = source.AgreedOn,
+                CommitmentsApprovedOn = source.CreatedOn
```

Comment:
Whyyyyy!??? ;) (Resharper maybe?)

I find it useful to have the commas on the trailing element as it eliminates future diffs on this line when adding more elements to the list.